src/frontend/screens/Onboarding/AddToProjectScreen/index.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from "react";
+import { NavigationStackScreenComponent } from "react-navigation-stack";
+import { View, StyleSheet } from "react-native";
+import { FormattedMessage, defineMessages } from "react-intl";
+
+import { MEDIUM_BLUE, WHITE } from "../../../lib/styles";
+import HeaderTitle from "../../../sharedComponents/HeaderTitle";
+import { BackIcon } from "../../../sharedComponents/icons";
+import IconButton from "../../../sharedComponents/IconButton";
+import { WithWifiBar } from "../WithWifiBar";
+
+import { ScanQrCodeStep } from "./ScanQrCodeStep";
+import { DeviceFoundStep } from "./DeviceFoundStep";
+import { SuccessStep } from "./SuccessStep";
+
+type Step = "scan" | "found" | "success";
```

Comment:
You don't have to change this, but if you make this an `enum` instead of a `type`, you will get better typesafety.

Reply:
yeah there's an everlasting struggle between when to use `type` vs `enum` 😄 I've generally avoided enums because of some TS-specific gotchas and the fact that they generate some excess code after compilation, but I don't have a particular aversion to it otherwise